app/src/main/res/layout/promoted_story_card.xml

```diff
@@ -45,7 +42,7 @@
         app:lessonThumbnail="@{viewModel.promotedStory.lessonThumbnail}" />
 
       <LinearLayout
-        android:layout_width="match_parent"
+        android:layout_width="0dp"
```

Comment:
Why is this changed correct? It seems unrelated compared to other fixes in this PR.